kube-scheduler-healthcheck

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -eu
+TEST_POD_NAMESPACE="${TEST_POD_NAMESPACE:-kube-system}"
+TEST_POD_IMAGE="${TEST_POD_IMAGE:-gcr.io/google_containers/pause:3.0.0}"
+
+cat > /tmp/pod.yaml <<endOfPodDef
+apiVersion: v1
+kind: Pod
+metadata:
+  name: scheduler-pod-healthcheck
+spec:
+  containers:
+  - name: pause
+    image: ${TEST_POD_IMAGE}
+endOfPodDef
+
+kubectl delete pod scheduler-pod-healthcheck -n $TEST_POD_NAMESPACE --request-timeout=30s || true
```

Comment:
declare an alias or variable to reduce the duplication

Reply:
for the whole line?

Reply:
for `kubectl --namespace $TEST_POD_NAMESPACE --request-timeout=30s`